source/containers.tex

```diff
@@ -3236,9 +3237,11 @@
 \throws Nothing unless one of the element-wise swap calls throws an exception.
 
 \pnum
-\realnote Unlike the \tcode{swap} function for other containers, \tcode{array::swap}
+\begin{note}
+Unlike the \tcode{swap} function for other containers, \tcode{array::swap}
 takes linear time, may exit via an exception, and does not cause iterators to
 become associated with the other container.
```

Comment:
All I have is Note A and Note B in [container.requirements.general] after the table.

Suggestion: This says "As if by swap_ranges" for the effects. Let's change that to "Equivalent to" and we get all the missing pieces for free, and can leave the note as a note.